editor/src/components/shared/project-components.ts

```diff
@@ -366,6 +375,19 @@ const fragmentElementsDescriptors: ComponentDescriptorsForFile = {
   },
 }
 
+const samplesDescriptors: ComponentDescriptorsForFile = {
+  sampleText: {
+    properties: {},
+    variants: [
+      {
+        insertMenuLabel: 'Sample text',
+        elementToInsert: jsxElementWithoutUID('span', [], [jsxTextBlock('Utopia rules')]),
```

Comment:
`'Utopia rules'` :D ahem maybe we want to workshop the inserted string, cc @maltenuhn

Reply:
updated to `Sample text` in https://github.com/concrete-utopia/utopia/pull/3550/commits/a92586564f411a37e4635db01efba3120e122db9